dolphinscheduler-master/pom.xml

```diff
@@ -67,11 +71,24 @@
                 </exclusion>
             </exclusions>
         </dependency>
-
         <dependency>
             <groupId>org.codehaus.janino</groupId>
             <artifactId>janino</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.hadoop</groupId>
+            <artifactId>hadoop-common</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>*</groupId>
```

Comment:
Is it still need to import this jar package with exclusion *?

Reply:
> Is it still need to import this jar package with exclusion *?

Master component only uses the class `ThreadUtil` in `Hadoop-common` itself, nothing in its transitive dependencies is used. note exclusion * only excludes everything `Hadoop-common` brings, not including `Hadoop-common` itself